facebook/delphiFacebook/R/contingency_run.R

```diff
@@ -197,28 +215,46 @@ run_contingency_tables <- function(params) {
 #' @importFrom lubridate ymd days
 run_contingency_tables_many_periods <- function(params, aggregations)
 {
-  if (!is.null(params$end_date) & !is.null(params$n_periods)) {
-    ## Produce historical CSVs
+  if (!is.null(params$n_periods)) {
+    msg_plain(paste0("Producing CSVs for ", params$n_periods, " time periods"))
 
     if (params$aggregate_range == "week") {
       period_step <- days(7)
     } else if (params$aggregate_range == "month") {
       period_step <- months(1)
     } else if (is.null(params$aggregate_range)) {
```

Comment:
maybe this should just be an `else`, because if `params$aggregate_range == "ducks"`, the `period_step` will never be set and this will go badly